tgui/packages/tgui/interfaces/TeleConsole.tsx

```diff
@@ -0,0 +1,196 @@
+              onClick={() => act('setY', { value: ytarget - 10 })}
+            />
+            <Button
+              icon="caret-left"
+              onClick={() => act('setY', { value: ytarget - 1 })}
+            />
+            <Button.Input
+              content={ytarget}
+              onCommit={(e, value) => act('setY', { value: value })}
+            />
+            <Button
+              icon="caret-right"
+              onClick={() => act('setY', { value: ytarget + 1 })}
+            />
+            <Button
+              icon="forward"
+              onClick={() => act('setY', { value: ytarget + 10 })}
+            />
+          </Box>
+          <Box>
+            {"Z: "}
+            <Button
+              icon="caret-left"
+              onClick={() => act('setZ', { value: ztarget - 1 })}
+            />
+            <Button.Input
+              content={ztarget}
+              onCommit={(e, value) => act('setZ', { value: value })}
+            />
+            <Button
+              icon="caret-right"
+              onClick={() => act('setZ', { value: ztarget + 1 })}
+            />
+          </Box>
+        </Section>
+        <Section width="80%" mx="auto">
+          <Box>
+            <Button
+              color="green"
+              icon="sign-out-alt"
+              onClick={() => act("send")}
+              content="Send"
+              disabled={!host_id}
+            />
+            <Button
+              color="green"
+              icon="sign-in-alt"
+              onClick={() => act("receive")}
+              content="Recieve"
+              disabled={!host_id}
+            />
+            <Button
+              color="green"
+              onClick={() => act("portal")}
+              disabled={!host_id}
+            >
+              <Icon name="ring" rotation={90} />Toggle Portal
+            </Button>
+          </Box>
+          <Button
+            color="green"
+            icon="magnifying-glass"
+            onClick={() => act("scan")}
+            content="Scan"
+            disabled={!host_id}
+          />
+        </Section>
+        <Section width="80%" mx="auto">
+          {"Bookmarks: "}
+          <Button
+            color="green"
+            icon="add"
+            onClick={() => act("addbookmark")}
+            disabled={bookmarks.length >= max_bookmarks}
+          />
+          {bookmarks.map(mark => {
+            return (
+              <Box key={mark["ref"]}>
+                <Button
+                  icon="bookmark"
+                  onClick={() => act("restorebookmark", { value: mark["ref"] })}
+                  content={mark["name"]}
+                />
+                {mark["xyz"]}
+                <Button
+                  icon="trash"
+                  color="red"
+                  onClick={() => act("deletebookmark", { value: mark["ref"] })}
+                />
+              </Box>
+            );
+          })}
+        </Section>
+        {panel_open ? (
```

Comment:
```suggestion
        {!!panel_open && (
```
Then you can drop the `: null` at the other end of this. `!!` is required because tgui does not pass booleans around as true/false, so this will probably render as `0` or something if you just leave as `panel_open && (`.